<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: CharacterBeforePHPOpeningTagStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Opening Tag at Start of File">
    <standard>
    <![CDATA[
    The opening php tag should be the first item in the file.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A file starting with an opening php tag.">
        <![CDATA[
<em></em><?php
echo 'Foo';
        ]]>
        </code>
        <code title="Invalid: A file with content before the opening php tag.">
        <![CDATA[
<em>Beginning content</em>
<?php
echo 'Foo';
        ]]>
        </code>
    </code_comparison>
</documentation>
