<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ForbiddenFunctionsStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Forbidden Functions">
    <standard>
    <![CDATA[
    The forbidden functions sizeof() and delete() should not be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: count() is used in place of sizeof().">
        <![CDATA[
$foo = <em>count</em>($bar);
        ]]>
        </code>
        <code title="Invalid: sizeof() is used.">
        <![CDATA[
$foo = <em>sizeof</em>($bar);
        ]]>
        </code>
    </code_comparison>
</documentation>
