<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: LowerCaseTypeStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Lowercase PHP Types">
    <standard>
    <![CDATA[
    All PHP types used for parameter type and return type declarations should be lowercase.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Lowercase type declarations used.">
        <![CDATA[
function myFunction(int $foo) : string {
}
        ]]>
        </code>
        <code title="Invalid: Non-lowercase type declarations used.">
        <![CDATA[
function myFunction(<em>Int</em> $foo) : <em>STRING</em> {
}
        ]]>
        </code>
    </code_comparison>
    <standard>
    <![CDATA[
    All PHP types used for type casting should be lowercase.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Lowercase type used.">
        <![CDATA[
$foo = (bool) $isValid;
        ]]>
        </code>
        <code title="Invalid: Non-lowercase type used.">
        <![CDATA[
$foo = <em>(BOOL)</em> $isValid;
        ]]>
        </code>
    </code_comparison>
</documentation>
