<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: LowerCaseKeywordStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Lowercase Keywords">
    <standard>
    <![CDATA[
    All PHP keywords should be lowercase.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Lowercase array keyword used.">
        <![CDATA[
$foo = <em>array</em>();
        ]]>
        </code>
        <code title="Invalid: Non-lowercase array keyword used.">
        <![CDATA[
$foo = <em>Array</em>();
        ]]>
        </code>
    </code_comparison>
</documentation>
