<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: NoSilencedErrorsStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Silenced Errors">
    <standard>
    <![CDATA[
    Suppressing Errors is not allowed.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: isset() is used to verify that a variable exists before trying to use it.">
        <![CDATA[
if (<em>isset($foo)</em> && $foo) {
    echo "Hello\n";
}
        ]]>
        </code>
        <code title="Invalid: Errors are suppressed.">
        <![CDATA[
if (<em>@</em>$foo) {
    echo "Hello\n";
}
        ]]>
        </code>
    </code_comparison>
</documentation>
