<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: SAPIUsageStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="SAPI Usage">
    <standard>
    <![CDATA[
    The PHP_SAPI constant should be used instead of php_sapi_name().
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: PHP_SAPI is used.">
        <![CDATA[
if (<em>PHP_SAPI</em> === 'cli') {
    echo "Hello, CLI user.";
}
        ]]>
        </code>
        <code title="Invalid: php_sapi_name() is used.">
        <![CDATA[
if (<em>php_sapi_name()</em> === 'cli') {
    echo "Hello, CLI user.";
}
        ]]>
        </code>
    </code_comparison>
</documentation>
