<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: UpperCaseConstantStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Uppercase PHP Constants">
    <standard>
    <![CDATA[
    The <em>true</em>, <em>false</em> and <em>null</em> constants must always be uppercase.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: uppercase constants">
        <![CDATA[
if ($var === <em>FALSE</em> || $var === <em>NULL</em>) {
    $var = <em>TRUE</em>;
}
        ]]>
        </code>
        <code title="Invalid: lowercase constants">
        <![CDATA[
if ($var === <em>false</em> || $var === <em>null</em>) {
    $var = <em>true</em>;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
