<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ClassDeclarationStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Class Declarations">
    <standard>
    <![CDATA[
    There should be exactly 1 space between the abstract or final keyword and the class keyword and between the class keyword and the class name.  The extends and implements keywords, if present, must be on the same line as the class name.  When interfaces implemented are spread over multiple lines, there should be exactly 1 interface mentioned per line indented by 1 level.  The closing brace of the class must go on the first line after the body of the class and must be on a line by itself.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Correct spacing around class keyword.">
        <![CDATA[
abstract<em> </em>class<em> </em>Foo
{
}
        ]]>
        </code>
        <code title="Invalid: 2 spaces used around class keyword.">
        <![CDATA[
abstract<em>  </em>class<em>  </em>Foo
{
}
        ]]>
        </code>
    </code_comparison>
</documentation>
