<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: PropertyDeclarationStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Property Declarations">
    <standard>
    <![CDATA[
    Property names should not be prefixed with an underscore to indicate visibility.  Visibility should be used to declare properties rather than the var keyword.  Only one property should be declared within a statement.  The static declaration must come after the visibility declaration.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Correct property naming.">
        <![CDATA[
class Foo
{
    private $<em>bar</em>;
}
        ]]>
        </code>
        <code title="Invalid: An underscore prefix used to indicate visibility.">
        <![CDATA[
class Foo
{
    private $<em>_bar</em>;
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Visibility of property declared.">
        <![CDATA[
class Foo
{
    <em>private</em> $bar;
}
        ]]>
        </code>
        <code title="Invalid: Var keyword used to declare property.">
        <![CDATA[
class Foo
{
    <em>var</em> $bar;
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: One property declared per statement.">
        <![CDATA[
class Foo
{
    private $bar;
    private $baz;
}
        ]]>
        </code>
        <code title="Invalid: Multiple properties declared in one statement.">
        <![CDATA[
class Foo
{
    private <em>$bar, $baz</em>;
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: If declared as static, the static declaration must come after the visibility declaration.">
        <![CDATA[
class Foo
{
    public <em>static</em> $bar;
    private $baz;
}
        ]]>
        </code>
        <code title="Invalid: Static declaration before the visibility declaration.">
        <![CDATA[
class Foo
{
    <em>static<em> protected $bar;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
