<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ValidVariableNameStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Variable Names">
    <standard>
    <![CDATA[
    Private member variable names should be prefixed with an underscore and public/protected variable names should not.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Proper member variable names.">
        <![CDATA[
class Foo
{
    public $<em>publicVar</em>;
    protected $<em>protectedVar</em>;
    private $<em>_privateVar</em>;
}
        ]]>
        </code>
        <code title="Invalid: underscores used on public/protected variables and not used on private variables.">
        <![CDATA[
class Foo
{
    public $<em>_publicVar</em>;
    protected $<em>_protectedVar</em>;
    private $<em>privateVar</em>;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
