<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ClosingTagStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Closing PHP Tags">
    <standard>
    <![CDATA[
    Files should not have closing php tags.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: No closing tag at the end of the file.">
        <![CDATA[
<?php
$var = 1;
        ]]>
        </code>
        <code title="Invalid: A closing php tag is included at the end of the file.">
        <![CDATA[
<?php
$var = 1;
<em>?></em>
        ]]>
        </code>
    </code_comparison>
</documentation>
