<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: InlineCommentStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Inline Comments">
    <standard>
    <![CDATA[
    Perl-style # comments are not allowed.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A // style comment.">
        <![CDATA[
<em>//</em> A comment.
        ]]>
        </code>
        <code title="Invalid: A # style comment.">
        <![CDATA[
<em>#</em> A comment.
        ]]>
        </code>
    </code_comparison>
</documentation>
