<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: InlineHTMLStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Inline HTML">
    <standard>
    <![CDATA[
    Files that contain php code should only have php code and should not have any "inline html".
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A php file with only php code in it.">
        <![CDATA[
<?php
$foo = 'bar';
echo $foo . 'baz';
        ]]>
        </code>
        <code title="Invalid: A php file with html in it outside of the php tags.">
        <![CDATA[
<em>some string here</em>
<?php
$foo = 'bar';
echo $foo . 'baz';
        ]]>
        </code>
    </code_comparison>
</documentation>
