<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: OneObjectStructurePerFileStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="One Object Structure Per File">
    <standard>
    <![CDATA[
    There should only be one class or interface or trait defined in a file.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Only one object structure in the file.">
        <![CDATA[
<?php
<em>trait Foo</em>
{
}
        ]]>
        </code>
        <code title="Invalid: Multiple object structures defined in one file.">
        <![CDATA[
<?php
<em>trait Foo</em>
{
}

<em>class Bar</em>
{
}
        ]]>
        </code>
    </code_comparison>
</documentation>
