<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: OneTraitPerFileStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="One Trait Per File">
    <standard>
    <![CDATA[
    There should only be one trait defined in a file.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Only one trait in the file.">
        <![CDATA[
<?php
<em>trait Foo</em>
{
}
        ]]>
        </code>
        <code title="Invalid: Multiple traits defined in one file.">
        <![CDATA[
<?php
<em>trait Foo</em>
{
}

<em>trait Bar</em>
{
}
        ]]>
        </code>
    </code_comparison>
</documentation>
