<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ClassDeclarationStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Class Declarations">
    <standard>
    <![CDATA[
    The opening brace of a class must be on the line after the definition by itself.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Opening brace on the correct line.">
        <![CDATA[
class Foo
<em>{</em>
}
        ]]>
        </code>
        <code title="Invalid: Opening brace on same line as declaration.">
        <![CDATA[
class Foo <em>{</em>
}
        ]]>
        </code>
    </code_comparison>
</documentation>
