<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: FunctionDuplicateArgumentStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Lowercase Built-In functions">
    <standard>
    <![CDATA[
    All PHP built-in functions should be lowercased when called.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Lowercase function call.">
        <![CDATA[
if (<em>isset</em>($foo)) {
    echo $foo;
}
]]>
        </code>
        <code title="Invalid: isset not called as lowercase.">
        <![CDATA[
if (<em>isSet</em>($foo)) {
    echo $foo;
}
]]>
        </code>
    </code_comparison>
</documentation>
