<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: LowercaseFunctionKeywordsStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Lowercase Function Keywords">
    <standard>
    <![CDATA[
    The php keywords function, public, private, protected, and static should be lowercase.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Lowercase function keyword.">
        <![CDATA[
<em>function</em> foo()
{
    return true;
}
]]>
        </code>
        <code title="Invalid: Uppercase function keyword.">
        <![CDATA[
<em>FUNCTION</em> foo()
{
    return true;
}
]]>
        </code>
    </code_comparison>
</documentation>
