<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ClosureLinterStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Closure Linter">
    <standard>
    <![CDATA[
    All javascript files should pass basic Closure Linter tests.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Valid JS Syntax is used.">
        <![CDATA[
var foo = [1, 2<em></em>];
        ]]>
        </code>
        <code title="Invalid: Trailing comma in a javascript array.">
        <![CDATA[
var foo = [1, 2<em>,</em>];
        ]]>
        </code>
    </code_comparison>
</documentation>
