<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: CSSLintStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="CSSLint">
    <standard>
    <![CDATA[
    All css files should pass the basic csslint tests.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Valid CSS Syntax is used.">
        <![CDATA[
.foo: { width: 100<em></em>%; }
        ]]>
        </code>
        <code title="Invalid: The CSS has a typo in it.">
        <![CDATA[
.foo: { width: 100<em> </em>%; }
        ]]>
        </code>
    </code_comparison>
</documentation>
