<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: JSHintStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="JSHint">
    <standard>
    <![CDATA[
    All javascript files should pass basic JSHint tests.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Valid JS Syntax is used.">
        <![CDATA[
<em>var</em> foo = 5;
        ]]>
        </code>
        <code title="Invalid: The Javascript is using an undefined variable.">
        <![CDATA[
<em></em>foo = 5;
        ]]>
        </code>
    </code_comparison>
</documentation>
