<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ClassDeclarationStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Class Declaration">
    <standard>
    <![CDATA[
    Each class must be in a file by itself and must be under a namespace (a top-level vendor name).
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: One class in a file.">
        <![CDATA[
<?php
namespace Foo;

<em>class Bar</em> {
}
        ]]>
        </code>
        <code title="Invalid: Multiple classes in a single file.">
        <![CDATA[
<?php
namespace Foo;

<em>class Bar</em> {
}

<em>class Baz</em> {
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: A vendor-level namespace is used.">
        <![CDATA[
<?php
<em>namespace Foo;</em>

class Bar {
}
        ]]>
        </code>
        <code title="Invalid: No namespace used in file.">
        <![CDATA[
<?php
class Bar {
}
        ]]>
        </code>
    </code_comparison>
</documentation>
