<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: CodeAnalyzerStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Zend Code Analyzer">
    <standard>
    <![CDATA[
    PHP Code should pass the zend code analyzer.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Valid PHP Code.">
        <![CDATA[
function foo($bar, $baz)
{
    return <em>$bar + $baz</em>;
}
        ]]>
        </code>
        <code title="Invalid: There is an unused function parameter.">
        <![CDATA[
function foo($bar, $baz)
{
    return <em>$bar + 2</em>;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
