<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: DisallowMultipleStatementsStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Multiple Statements On a Single Line">
    <standard>
    <![CDATA[
    Multiple statements are not allowed on a single line.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Two statements are spread out on two separate lines.">
        <![CDATA[
$foo = 1;
$bar = 2;
        ]]>
        </code>
        <code title="Invalid: Two statements are combined onto one line.">
        <![CDATA[
$foo = 1; $bar = 2;
        ]]>
        </code>
    </code_comparison>
</documentation>
