<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: NoSpaceAfterCastStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Space After Casts">
    <standard>
    <![CDATA[
    Spaces are not allowed after casting operators.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A cast operator is immediately before its value.">
        <![CDATA[
$foo = (string)1;
        ]]>
        </code>
        <code title="Invalid: A cast operator is followed by whitespace.">
        <![CDATA[
$foo = (string)<em> </em>1;
        ]]>
        </code>
    </code_comparison>
</documentation>
