<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: SpaceAfterNotStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Space After NOT operator">
    <standard>
    <![CDATA[
    Exactly one space is allowed after the NOT operator.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A NOT operator followed by one space.">
        <![CDATA[
if (!<em> </em>$someVar || !<em> </em>$x instanceOf stdClass) {};
        ]]>
        </code>
        <code title="Invalid: A NOT operator not followed by whitespace.">
        <![CDATA[
if (!<em></em>$someVar || !<em></em>$x instanceOf stdClass) {};
        ]]>
        </code>
        <code title="Invalid: A NOT operator followed by a new line or more than one space.">
        <![CDATA[
if (!<em>     </em>$someVar || !<em>
    </em>$x instanceOf stdClass) {};
        ]]>
        </code>
    </code_comparison>
</documentation>
