<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: CallTimePassByReferenceStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Call-Time Pass-By-Reference">
    <standard>
    <![CDATA[
    Call-time pass-by-reference is not allowed. It should be declared in the function definition.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Pass-by-reference is specified in the function definition.">
        <![CDATA[
function foo(<em>&</em>$bar)
{
    $bar++;
}

$baz = 1;
foo($baz);
        ]]>
        </code>
        <code title="Invalid: Pass-by-reference is done in the call to a function.">
        <![CDATA[
function foo($bar)
{
    $bar++;
}

$baz = 1;
foo(<em>&</em>$baz);
        ]]>
        </code>
    </code_comparison>
</documentation>
