<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: FunctionCallArgumentSpacingStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Function Argument Spacing">
    <standard>
    <![CDATA[
    Function arguments should have one space after a comma, and single spaces surrounding the equals sign for default values.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Single spaces after a comma.">
        <![CDATA[
function foo($bar,<em> </em>$baz)
{
}
        ]]>
        </code>
        <code title="Invalid: No spaces after a comma.">
        <![CDATA[
function foo($bar,<em></em>$baz)
{
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Single spaces around an equals sign in function declaration.">
        <![CDATA[
function foo($bar, $baz<em> </em>=<em> </em>true)
{
}
        ]]>
        </code>
        <code title="Invalid: No spaces around an equals sign in function declaration.">
        <![CDATA[
function foo($bar, $baz<em></em>=<em></em>true)
{
}
        ]]>
        </code>
    </code_comparison>
</documentation>
