<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: CastSpacingStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Cast Whitespace">
    <standard>
    <![CDATA[
    Casts should not have whitespace inside the parentheses.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: No spaces.">
        <![CDATA[
$foo = (<em></em>int<em></em>)'42';
]]>
        </code>
        <code title="Invalid: Whitespace used inside parentheses.">
        <![CDATA[
$foo = (<em> </em>int<em> </em>)'42';
]]>
        </code>
    </code_comparison>
</documentation>
