<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: FunctionOpeningBraceStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Function Opening Brace Whitespace">
    <standard>
    <![CDATA[
    The opening brace for functions should be on a new line with no blank lines surrounding it.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Opening brace is on a new line.">
        <![CDATA[
function foo()
<em>{</em>
}
]]>
        </code>
        <code title="Invalid: Opening brace is on the same line as the function declaration.">
        <![CDATA[
function foo() <em>{</em>
}
]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: No blank lines after the opening brace.">
        <![CDATA[
function foo()
{
<em>    return 42;</em>
}
]]>
        </code>
        <code title="Invalid: A blank line after the opening brace.">
        <![CDATA[
function foo()
{
<em></em>
    return 42;
}
]]>
        </code>
    </code_comparison>
</documentation>
