<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: LanguageConstructSpacingStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Language Construct Whitespace">
    <standard>
    <![CDATA[
    The php constructs echo, print, return, include, include_once, require, require_once, and new should have one space after them.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: echo statement with a single space after it.">
        <![CDATA[
echo<em> </em>"hi";
]]>
        </code>
        <code title="Invalid: echo statement with no space after it.">
        <![CDATA[
echo<em></em>"hi";
]]>
        </code>
    </code_comparison>
</documentation>
