<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ObjectOperatorSpacingStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Object Operator Spacing">
    <standard>
    <![CDATA[
    The object operator (->) should not have any space around it.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: No spaces around the object operator.">
        <![CDATA[
$foo<em></em>-><em></em>bar();
]]>
        </code>
        <code title="Invalid: Whitespace surrounding the object operator.">
        <![CDATA[
$foo<em> </em>-><em> </em>bar();
]]>
        </code>
    </code_comparison>
</documentation>
