<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ScopeKeywordSpacingStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Scope Keyword Spacing">
    <standard>
    <![CDATA[
    The php keywords static, public, private, and protected should have one space after them.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A single space following the keywords.">
        <![CDATA[
public<em> </em>static<em> </em>function foo()
{
}
]]>
        </code>
        <code title="Invalid: Multiple spaces following the keywords.">
        <![CDATA[
public<em>  </em>static<em>  </em>function foo()
{
}
]]>
        </code>
    </code_comparison>
</documentation>
