<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: DocCommentAlignmentStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Doc Comment Alignment">
    <standard>
    <![CDATA[
    The asterisks in a doc comment should align, and there should be one space between the asterisk and tags.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Asterisks are aligned.">
        <![CDATA[
/**
<em> </em>* @see foo()
<em> </em>*/
]]>
        </code>
        <code title="Invalid: Asterisks are not aligned.">
        <![CDATA[
/**
<em>  </em>* @see foo()
<em></em>*/
]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: One space between asterisk and tag.">
        <![CDATA[
/**
 *<em> </em>@see foo()
 */
]]>
        </code>
        <code title="Invalid: Incorrect spacing used.">
        <![CDATA[
/**
 *<em>  </em>@see foo()
 */
]]>
        </code>
    </code_comparison>
</documentation>
