<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: FunctionCommentThrowTagStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Doc Comment Throws Tag">
    <standard>
    <![CDATA[
    If a function throws any exceptions, they should be documented in a @throws tag.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: @throws tag used.">
        <![CDATA[
/**
 * <em>@throws Exception all the time</em>
 * @return void
 */
function foo()
{
    throw new Exception('Danger!');
}
]]>
        </code>
        <code title="Invalid: No @throws tag used for throwing function.">
        <![CDATA[
/**
 * @return void
 */
function foo()
{
    throw new Exception('Danger!');
}
]]>
        </code>
    </code_comparison>
</documentation>
