<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: NamespaceDeclarationStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Namespace Declarations">
    <standard>
    <![CDATA[
    There must be one blank line after the namespace declaration.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: One blank line after the namespace declaration.">
        <![CDATA[
namespace \Foo\Bar;
<em></em>
use \Baz;
        ]]>
        </code>
        <code title="Invalid: No blank line after the namespace declaration.">
        <![CDATA[
namespace \Foo\Bar;
use \Baz;
        ]]>
        </code>
    </code_comparison>
</documentation>
