<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: AssignmentInConditionStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Assignment In Condition">
    <standard>
    <![CDATA[
    Variable assignments should not be made within conditions.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A variable comparison being executed within a condition.">
        <![CDATA[
if (<em>$test === 'abc'</em>) {
    // Code.
}
        ]]>
        </code>
        <code title="Invalid: A variable assignment being made within a condition.">
        <![CDATA[
if (<em>$test = 'abc'</em>) {
    // Code.
}
        ]]>
        </code>
    </code_comparison>
</documentation>
