<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: EmptyStatementStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Empty Statements">
    <standard>
    <![CDATA[
    Control Structures must have at least one statement inside of the body.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: There is a statement inside the control structure.">
        <![CDATA[
if ($test) {
    $var = 1;
}
        ]]>
        </code>
        <code title="Invalid: The control structure has no statements.">
        <![CDATA[
if ($test) {
    <em>// do nothing</em>
}
        ]]>
        </code>
    </code_comparison>
</documentation>
