<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ForLoopShouldBeWhileLoopStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Condition-Only For Loops">
    <standard>
    <![CDATA[
    For loops that have only a second expression (the condition) should be converted to while loops.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A for loop is used with all three expressions.">
        <![CDATA[
for (<em>$i = 0</em>; $i < 10; <em>$i++</em>) {
    echo "{$i}\n";
}
        ]]>
        </code>
        <code title="Invalid: A for loop is used without a first or third expression.">
        <![CDATA[
for (<em></em>;$test;<em></em>) {
    $test = doSomething();
}
        ]]>
        </code>
    </code_comparison>
</documentation>
