<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: JumbledIncrementerStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Jumbled Incrementers">
    <standard>
    <![CDATA[
    Incrementers in nested loops should use different variable names.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Two different variables being used to increment.">
        <![CDATA[
for ($i = 0; $i < 10; <em>$i++</em>) {
    for ($j = 0; $j < 10; <em>$j++</em>) {
    }
}
        ]]>
        </code>
        <code title="Invalid: Inner incrementer is the same variable name as the outer one.">
        <![CDATA[
for ($i = 0; $i < 10; <em>$i++</em>) {
    for ($j = 0; $j < 10; <em>$i++</em>) {
    }
}
        ]]>
        </code>
    </code_comparison>
</documentation>
