<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: UnconditionalIfStatementStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Unconditional If Statements">
    <standard>
    <![CDATA[
    If statements that are always evaluated should not be used.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: An if statement that only executes conditionally.">
        <![CDATA[
if (<em>$test</em>) {
    $var = 1;
}
        ]]>
        </code>
        <code title="Invalid: An if statement that is always performed.">
        <![CDATA[
if (<em>true</em>) {
    $var = 1;
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: An if statement that only executes conditionally.">
        <![CDATA[
if (<em>$test</em>) {
    $var = 1;
}
        ]]>
        </code>
        <code title="Invalid: An if statement that is never performed.">
        <![CDATA[
if (<em>false</em>) {
    $var = 1;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
