<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: UnnecessaryFinalModifierStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Unnecessary Final Modifiers">
    <standard>
    <![CDATA[
    Methods should not be declared final inside of classes that are declared final.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A method in a final class is not marked final.">
        <![CDATA[
final class Foo
{
    public function bar()
    {
    }
}
        ]]>
        </code>
        <code title="Invalid: A method in a final class is also marked final.">
        <![CDATA[
final class Foo
{
    public <em>final</em> function bar()
    {
    }
}
        ]]>
        </code>
    </code_comparison>
</documentation>
