<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: UnusedFunctionParameterStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Unused function parameters">
    <standard>
    <![CDATA[
    All parameters in a functions signature should be used within the function.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: All the parameters are used.">
        <![CDATA[
function addThree($a, $b, $c)
{
    return <em>$a + $b + $c</em>;
}
        ]]>
        </code>
        <code title="Invalid: One of the parameters is not being used.">
        <![CDATA[
function addThree($a, $b, $c)
{
    return <em>$a + $b</em>;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
