<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: UselessOverridingMethodStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Useless Overriding Methods">
    <standard>
    <![CDATA[
    Methods should not be defined that only call the parent method.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A method that extends functionality on a parent method.">
        <![CDATA[
final class Foo
{
    public function bar()
    {
        parent::bar();
        <em>$this->doSomethingElse();</em>
    }
}
        ]]>
        </code>
        <code title="Invalid: An overriding method that only calls the parent.">
        <![CDATA[
final class Foo
{
    public function bar()
    {
        <em>parent::bar();</em>
    }
}
        ]]>
        </code>
    </code_comparison>
</documentation>
