<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: MethodDeclarationStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Method Declarations">
    <standard>
    <![CDATA[
    Method names should not be prefixed with an underscore to indicate visibility.  The static keyword, when present, should come after the visibility declaration, and the final and abstract keywords should come before.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Correct method naming.">
        <![CDATA[
class Foo
{
    private function <em>bar</em>()
    {
    }
}
        ]]>
        </code>
        <code title="Invalid: An underscore prefix used to indicate visibility.">
        <![CDATA[
class Foo
{
    private function <em>_bar</em>()
    {
    }
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Correct ordering of method prefixes.">
        <![CDATA[
class Foo
{
    <em>final public static</em> function <em>bar</em>()
    {
    }
}
        ]]>
        </code>
        <code title="Invalid: static keyword used before visibility and final used after.">
        <![CDATA[
class Foo
{
    <em>static public final</em> function <em>bar</em>()
    {
    }
}
        ]]>
        </code>
    </code_comparison>
</documentation>
