<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: NullableTypeDeclarationStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Nullable Type Declarations Functions">
    <standard>
        <![CDATA[
    In nullable type declarations there MUST NOT be a space between the question mark and the type.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: no whitespace used.">
            <![CDATA[
public function functionName(
    ?string $arg1,
    ?int $arg2
): ?string {
}
        ]]>
        </code>
        <code title="Invalid: superfluous whitespace used.">
        <![CDATA[
public function functionName(
    ? string $arg1,
    ? int $arg2
): ? string {
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: no unexpected characters.">
            <![CDATA[
public function foo(?int $arg): ?string
{
}
        ]]>
        </code>
        <code title="Invalid: unexpected characters used.">
            <![CDATA[
public function bar(? /* comment */ int $arg): ?
    // nullable for a reason
    string
{
}
        ]]>
        </code>
    </code_comparison>
</documentation>
