<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ObjectOperatorIndentStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Object Operator Indentation">
    <standard>
    <![CDATA[
    Chained object operators when spread out over multiple lines should be the first thing on the line and be indented by 1 level.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Object operator at the start of a new line.">
        <![CDATA[
$foo
    <em>-></em>bar()
    <em>-></em>baz();
        ]]>
        </code>
        <code title="Invalid: Object operator at the end of the line.">
        <![CDATA[
$foo<em>-></em>
    bar()<em>-></em>
    baz();
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Object operator indented correctly.">
        <![CDATA[
$foo
<em>    </em>->bar()
<em>    </em>->baz();
        ]]>
        </code>
        <code title="Invalid: Object operator not indented correctly.">
        <![CDATA[
$foo
<em></em>->bar()
<em></em>->baz();
        ]]>
        </code>
    </code_comparison>
</documentation>
