<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ScopeIndentStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Scope Indentation">
    <standard>
    <![CDATA[
    Any scope openers except for switch statements should be indented 1 level.  This includes classes, functions, and control structures.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Consistent indentation level for scope.">
        <![CDATA[
function foo()
{
<em>    </em>if ($test) {
<em>        </em>$var = 1;
<em>    </em>}
}
        ]]>
        </code>
        <code title="Invalid: Indentation is not used for scope.">
        <![CDATA[
function foo()
{
<em></em>if ($test) {
<em></em>$var = 1;
<em></em>}
}
        ]]>
        </code>
    </code_comparison>
</documentation>
