<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ScopeClosingBraceStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Closing Brace Indentation">
    <standard>
    <![CDATA[
    Closing braces should be indented at the same level as the beginning of the scope.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Consistent indentation level for scope.">
        <![CDATA[
if ($test) {
    $var = 1;
}
        ]]>
        </code>
        <code title="Invalid: The ending brace is indented further than the if statement.">
        <![CDATA[
if ($test) {
    $var = 1;
<em>    </em>}
        ]]>
        </code>
    </code_comparison>
</documentation>
