<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: MultiLineConditionStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Multi-line If Conditions">
    <standard>
    <![CDATA[
    Multi-line if conditions should be indented one level and each line should begin with a boolean operator.  The end parenthesis should be on a new line.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Correct indentation.">
        <![CDATA[
if ($foo
<em>    </em>&& $bar
) {
}
        ]]>
        </code>
        <code title="Invalid: No indentation used on the condition lines.">
        <![CDATA[
if ($foo
<em></em>&& $bar
) {
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Boolean operator at the start of the line.">
        <![CDATA[
if ($foo
    <em>&&</em> $bar
) {
}
        ]]>
        </code>
        <code title="Invalid: Boolean operator at the end of the line.">
        <![CDATA[
if ($foo <em>&&</em>
    $bar
) {
}
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: End parenthesis on a new line.">
        <![CDATA[
if ($foo
    && $bar
<em>)</em> {
}
        ]]>
        </code>
        <code title="Invalid: End parenthesis not moved to a new line.">
        <![CDATA[
if ($foo
    && $bar<em>)</em> {
}
        ]]>
        </code>
    </code_comparison>
</documentation>
