<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ValidVariableNameStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Variable Names">
    <standard>
    <![CDATA[
    Variable names should be camelCased with the first letter lowercase.  Private and protected member variables should begin with an underscore
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A multi-word variable uses camel casing.">
        <![CDATA[
<em>$testNumber</em> = 1;
        ]]>
        </code>
        <code title="Invalid: A multi-word variable uses underscores and initial capitalization.">
        <![CDATA[
<em>$Test_Number</em> = 1;
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: A private member variable begins with an underscore.">
        <![CDATA[
class Foo
{
    private $<em>_</em>bar;
}
        ]]>
        </code>
        <code title="Invalid: A private member variable does not begin with an underscore.">
        <![CDATA[
class Foo
{
    private $bar;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
