<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: DisallowShortArraySyntaxStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Long Array Syntax">
    <standard>
    <![CDATA[
    Long array syntax must be used to define arrays.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Long form of array.">
        <![CDATA[
$arr = <em>array(</em>
    'foo' => 'bar',
<em>)</em>;
        ]]>
        </code>
        <code title="Invalid: Short form of array.">
        <![CDATA[
$arr = <em>[</em>
    'foo' => 'bar',
<em>]</em>;
        ]]>
        </code>
    </code_comparison>
</documentation>
