<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ClassInstantiationStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Class Instantiation">
    <standard>
    <![CDATA[
    When instantiating a new class, parenthesis MUST always be present even when there are no arguments passed to the constructor.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Parenthesis used.">
        <![CDATA[
new Foo();
        ]]>
        </code>
        <code title="Invalid: Parenthesis not used.">
        <![CDATA[
new Foo;
        ]]>
        </code>
    </code_comparison>
</documentation>
