<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ControlStructureSpacingStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Control Structure Spacing">
    <standard>
    <![CDATA[
    Control Structures should have 0 spaces after opening parentheses and 0 spaces before closing parentheses.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Correct spacing.">
        <![CDATA[
if (<em></em>$foo<em></em>) {
    $var = 1;
}
        ]]>
        </code>
        <code title="Invalid: Whitespace used inside the parentheses.">
        <![CDATA[
if (<em> </em>$foo<em> </em>) {
    $var = 1;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
