<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ElseIfDeclarationStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Elseif Declarations">
    <standard>
    <![CDATA[
    PHP's elseif keyword should be used instead of else if.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Single word elseif keyword used.">
        <![CDATA[
if ($foo) {
    $var = 1;
} <em>elseif</em> ($bar) {
    $var = 2;
}
        ]]>
        </code>
        <code title="Invalid: Separate else and if keywords used.">
        <![CDATA[
if ($foo) {
    $var = 1;
} <em>else if</em> ($bar) {
    $var = 2;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
