<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ArbitraryParenthesesSpacingStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Arbitrary Parentheses Spacing">
    <standard>
    <![CDATA[
    Arbitrary sets of parentheses should have no spaces inside.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: no spaces on the inside of a set of arbitrary parentheses.">
        <![CDATA[
$a = (null !== $extra);
        ]]>
        </code>
        <code title="Invalid: spaces on the inside of a set of arbitrary parentheses.">
        <![CDATA[
$a = ( null !== $extra );
        ]]>
        </code>
        <code title="Invalid: new lines on the inside of a set of arbitrary parentheses.">
        <![CDATA[
$a = (
    null !== $extra
);
        ]]>
        </code>
    </code_comparison>
</documentation>
