<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ScopeIndentStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Scope Indentation">
    <standard>
    <![CDATA[
    Indentation for control structures, classes, and functions should be 4 spaces per level.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: 4 spaces are used to indent a control structure.">
        <![CDATA[
if ($test) {
<em>    </em>$var = 1;
}
        ]]>
        </code>
        <code title="Invalid: 8 spaces are used to indent a control structure.">
        <![CDATA[
if ($test) {
<em>        </em>$var = 1;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
