<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: SpreadOperatorSpacingAfterStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Spacing After Spread Operator">
    <standard>
    <![CDATA[
    There should be no space between the spread operator and the variable/function call it applies to.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: No space between the spread operator and the variable/function call it applies to.">
        <![CDATA[
function foo(<em>&...$spread</em>) {
    bar(<em>...$spread</em>);

    bar(
        [<em>...$foo</em>],
        <em>...array_values($keyedArray)</em>
    );
}
        ]]>
        </code>
        <code title="Invalid: space found between the spread operator and the variable/function call it applies to.">
        <![CDATA[
function bar(<em>... </em>$spread) {
    bar(<em>...
        </em>$spread
    );

    bar(
        [<em>...  </em>$foo ],<em>.../*comment*/</em>array_values($keyedArray)
    );
}
        ]]>
        </code>
    </code_comparison>
</documentation>
