<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: MultiLineAssignmentStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Multi-Line Assignment">
    <standard>
    <![CDATA[
    Multi-line assignment should have the equals sign be the first item on the second line indented correctly.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Assignment operator at the start of the second line.">
        <![CDATA[
$foo
    <em>=</em> $bar;
        ]]>
        </code>
        <code title="Invalid: Assignment operator at end of first line.">
        <![CDATA[
$foo <em>=</em>
    $bar;
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Assignment operator indented one level.">
        <![CDATA[
$foo
<em>    </em>= $bar;
        ]]>
        </code>
        <code title="Invalid: Assignment operator not indented.">
        <![CDATA[
$foo
<em></em>= $bar;
        ]]>
        </code>
    </code_comparison>
</documentation>
