<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: EchoedStringsStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Echoed Strings">
    <standard>
    <![CDATA[
    Simple strings should not be enclosed in parentheses when being echoed.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Using echo without parentheses.">
        <![CDATA[
echo<em> </em>"Hello";
]]>
        </code>
        <code title="Invalid: Using echo with parentheses.">
        <![CDATA[
echo<em>(</em>"Hello"<em>)</em>;
]]>
        </code>
    </code_comparison>
</documentation>
