<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: CamelCapsFunctionNameStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="camelCaps Function Names">
    <standard>
    <![CDATA[
    Functions should use camelCaps format for their names. Only PHP's magic methods should use a double underscore prefix.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A function in camelCaps format.">
        <![CDATA[
function <em>doSomething</em>()
{
}
        ]]>
        </code>
        <code title="Invalid: A function in snake_case format.">
        <![CDATA[
function <em>do_something</em>()
{
}
        ]]>
        </code>
    </code_comparison>
</documentation>
