<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ConstructorNameStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Constructor name">
    <standard>
    <![CDATA[
    Constructors should be named __construct, not after the class.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: The constructor is named __construct.">
        <![CDATA[
class Foo
{
    function <em>__construct</em>()
    {
    }
}
        ]]>
        </code>
        <code title="Invalid: The old style class name constructor is used.">
        <![CDATA[
class Foo
{
    function <em>Foo</em>()
    {
    }
}
        ]]>
        </code>
    </code_comparison>
</documentation>
