<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: UpperCaseConstantNameStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Constant Names">
    <standard>
    <![CDATA[
     Constants should always be all-uppercase, with underscores to separate words.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: all uppercase">
        <![CDATA[
define('<em>FOO_CONSTANT</em>', 'foo');

class FooClass
{
    const <em>FOO_CONSTANT</em> = 'foo';
}
        ]]>
        </code>
        <code title="Invalid: mixed case">
        <![CDATA[
define('<em>Foo_Constant</em>', 'foo');

class FooClass
{
    const <em>foo_constant</em> = 'foo';
}
        ]]>
        </code>
    </code_comparison>
</documentation>
