<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: DuplicateClassNameStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Duplicate Class Names">
    <standard>
    <![CDATA[
    Class and Interface names should be unique in a project.  They should never be duplicated.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A unique class name.">
        <![CDATA[
class <em>Foo</em>
{
}
        ]]>
        </code>
        <code title="Invalid: A class duplicated (including across multiple files).">
        <![CDATA[
class <em>Foo</em>
{
}

class <em>Foo</em>
{
}
        ]]>
        </code>
    </code_comparison>
</documentation>
