<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: OpeningBraceSameLineStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Opening Brace on Same Line">
    <standard>
    <![CDATA[
    The opening brace of a class must be on the same line after the definition and must be the last thing on that line.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Opening brace on the same line.">
        <![CDATA[
class Foo <em>{</em>
}
        ]]>
        </code>
        <code title="Invalid: Opening brace on the next line.">
        <![CDATA[
class Foo
<em>{</em>
}
        ]]>
        </code>
        <code title="Invalid: Opening brace not last thing on the line.">
        <![CDATA[
class Foo {<em> // Start of class.</em>
}
        ]]>
        </code>
    </code_comparison>
</documentation>
