<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: IncludingFileStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Including Code">
    <standard>
    <![CDATA[
    Anywhere you are unconditionally including a class file, use <em>require_once</em>. Anywhere you are conditionally including a class file (for example, factory methods), use <em>include_once</em>. Either of these will ensure that class files are included only once. They share the same file list, so you don't need to worry about mixing them - a file included with <em>require_once</em> will not be included again by <em>include_once</em>.
    ]]>
    </standard>
    <standard>
    <![CDATA[
    Note that <em>include_once</em> and <em>require_once</em> are statements, not functions. Parentheses should not surround the subject filename.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: used as statement">
        <![CDATA[
require_once 'PHP/CodeSniffer.php';
        ]]>
        </code>
        <code title="Invalid: used as function">
        <![CDATA[
require_once<em>(</em>'PHP/CodeSniffer.php'<em>)</em>;
        ]]>
        </code>
    </code_comparison>
</documentation>
