<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ShortFormTypeKeywordsStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Short Form Type Keywords">
    <standard>
    <![CDATA[
    Short form of type keywords MUST be used i.e. bool instead of boolean, int instead of integer etc.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Short form type used.">
        <![CDATA[
$foo = (bool) $isValid;
        ]]>
        </code>
        <code title="Invalid: Long form type type used.">
        <![CDATA[
$foo = <em>(boolean)</em> $isValid;
        ]]>
        </code>
    </code_comparison>
</documentation>
