<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: StaticThisUsageStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Static This Usage">
    <standard>
    <![CDATA[
    Static methods should not use $this.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Using self:: to access static variables.">
        <![CDATA[
class Foo
{
    <em>static</em> function bar()
    {
        return <em>self</em>::$staticMember;
    }
}
]]>
        </code>
        <code title="Invalid: Using $this-> to access static variables.">
        <![CDATA[
class Foo
{
    <em>static</em> function bar()
    {
    return <em>$this</em>->$staticMember;
    }
}
]]>
        </code>
    </code_comparison>
</documentation>
