<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: DisallowYodaConditionsStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Disallow Yoda conditions">
    <standard>
    <![CDATA[
    Yoda conditions are disallowed.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: value to be asserted must go on the right side of the comparison.">
        <![CDATA[
if ($test === null) <em>{</em>
    $var = 1;
<em>}</em>
        ]]>
        </code>
        <code title="Invalid: value to be asserted must not be on the left.">
        <![CDATA[
if (null === $test) <em>{</em>
    $var = 1;
<em>}</em>
        ]]>
        </code>
    </code_comparison>
</documentation>
