<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: InlineControlStructureStandard.xml
  ~  Last Modified: 18.06.22 Ğ³., 10:21 Ñ‡.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Inline Control Structures">
    <standard>
    <![CDATA[
    Control Structures should use braces.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Braces are used around the control structure.">
        <![CDATA[
if ($test) <em>{</em>
    $var = 1;
<em>}</em>
        ]]>
        </code>
        <code title="Invalid: No braces are used for the control structure..">
        <![CDATA[
if ($test)
    $var = 1;
        ]]>
        </code>
    </code_comparison>
</documentation>
