<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: FixmeStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Fixme Comments">
    <standard>
    <![CDATA[
    FIXME Statements should be taken care of.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A comment without a fixme.">
        <![CDATA[
// <em>Handle strange case</em>
if ($test) {
    $var = 1;
}
        ]]>
        </code>
        <code title="Invalid: A fixme comment.">
        <![CDATA[
// <em>FIXME</em>: This needs to be fixed!
if ($test) {
    $var = 1;
}
        ]]>
        </code>
    </code_comparison>
</documentation>
