<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: LowercaseClassKeywordsStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Lowercase Class Keywords">
    <standard>
    <![CDATA[
    The php keywords class, interface, trait, extends, implements, abstract, final, var, and const should be lowercase.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Lowercase class keywords.">
        <![CDATA[
<em>final</em> <em>class</em> Foo <em>extends</em> Bar
{
}
]]>
        </code>
        <code title="Invalid: Initial capitalization of class keywords.">
        <![CDATA[
<em>Final</em> <em>Class</em> Foo <em>Extends</em> Bar
{
}
]]>
        </code>
    </code_comparison>
</documentation>
