<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ForEachLoopDeclarationStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Foreach Loop Declarations">
    <standard>
    <![CDATA[
    There should be a space between each element of a foreach loop and the as keyword should be lowercase.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Correct spacing used.">
        <![CDATA[
foreach (<em></em>$foo<em> </em>as<em> </em>$bar<em> </em>=><em> </em>$baz<em></em>) {
    echo $baz;
}
]]>
        </code>
        <code title="Invalid: Invalid spacing used.">
        <![CDATA[
foreach (<em> </em>$foo<em>  </em>as<em>  </em>$bar<em></em>=><em></em>$baz<em> </em>) {
    echo $baz;
}
]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Lowercase as keyword.">
        <![CDATA[
foreach ($foo <em>as</em> $bar => $baz) {
    echo $baz;
}
]]>
        </code>
        <code title="Invalid: Uppercase as keyword.">
        <![CDATA[
foreach ($foo <em>AS</em> $bar => $baz) {
    echo $baz;
}
]]>
        </code>
    </code_comparison>
</documentation>
