<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: ForLoopDeclarationStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="For Loop Declarations">
    <standard>
    <![CDATA[
    In a for loop declaration, there should be no space inside the brackets and there should be 0 spaces before and 1 space after semicolons.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Correct spacing used.">
        <![CDATA[
for (<em></em>$i = 0; $i < 10; $i++<em></em>) {
    echo $i;
}
]]>
        </code>
        <code title="Invalid: Invalid spacing used inside brackets.">
        <![CDATA[
for (<em> </em>$i = 0; $i < 10; $i++<em> </em>) {
    echo $i;
}
]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Correct spacing used.">
        <![CDATA[
for ($i = 0<em></em>; $i < 10<em></em>; $i++) {
    echo $i;
}
]]>
        </code>
        <code title="Invalid: Invalid spacing used before semicolons.">
        <![CDATA[
for ($i = 0<em> </em>; $i < 10<em> </em>; $i++) {
    echo $i;
}
]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Correct spacing used.">
        <![CDATA[
for ($i = 0;<em> </em>$i < 10;<em> </em>$i++) {
    echo $i;
}
]]>
        </code>
        <code title="Invalid: Invalid spacing used after semicolons.">
        <![CDATA[
for ($i = 0;<em></em>$i < 10;<em></em>$i++) {
    echo $i;
}
]]>
        </code>
    </code_comparison>
</documentation>
