<!--
  ~
  ~ Simple password manager written in PHP with Bootstrap and PDO database connections
  ~
  ~  File name: SideEffectsStandard.xml
  ~  Last Modified: 18.06.22 г., 10:21 ч.
  ~
  ~  @link          https://blacktiehost.com
  ~  @since         1.0.0
  ~  @version       2.2.0
  ~  @author        Milen Karaganski <milen@blacktiehost.com>
  ~
  ~  @license       GPL-3.0+
  ~  @license       http://www.gnu.org/licenses/gpl-3.0.txt
  ~  @copyright     Copyright (c)  2020 - 2022 blacktiehost.com
  ~
  -->

<documentation title="Side Effects">
    <standard>
    <![CDATA[
    A php file should either contain declarations with no side effects, or should just have logic (including side effects) with no declarations.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: A class defined in a file by itself.">
        <![CDATA[
<?php
class Foo
{
}
        ]]>
        </code>
        <code title="Invalid: A class defined in a file with other code.">
        <![CDATA[
<?php
class Foo
{
}

<em>echo "Class Foo loaded."</em>
        ]]>
        </code>
    </code_comparison>
</documentation>
